public/bd.json

```diff
@@ -0,0 +1,68 @@
+{
+  "users": [
+    {
+      "id": "0",
+      "name": "Ivan",
+      "direction": "Frontend",
+      "email": "qwe@qwe.com",
+      "lastName": "Ivanov",
+      "sex": "Male",
+      "education": "BSUIR",
+      "birthDate": "07.08.1992",
+      "universityAverageAcore": "8.5",
+      "mathScore": "8",
+      "address": "Minsk",
+      "mobilePhone": "+37529-1414144",
+      "skype": "ivanSkype",
+      "startDate": "16.01.2022",
+      "tasksId": ["0"]
+    },
+    {
+      "id": "1",
+      "name": "Olya",
+      "direction": "Java",
+      "email": "asd@asd.com",
+      "lastName": "Ivanova",
+      "sex": "Woman",
+      "education": "BGU",
+      "birthDate": "05.10.1998",
+      "universityAverageAcore": "7.8",
+      "mathScore": "9",
+      "address": "Gomel",
+      "mobilePhone": "+37529-2222222",
+      "skype": "olyaSkype",
+      "startDate": "15.01.2022",
+      "tasksId": ["0", "1"]
+    }
+  ],
+  "tasks": [
+    {
+      "id": "0",
+      "name": "Creat Button",
+      "description": "Color red",
+      "startDate": "16.02.2022",
+      "deadlineDate": "25.02.2022",
+      "status": "active",
+      "track": {"data": "25.02.2022", "node": "node Taks 1"}
+    },
+    {
+      "id": "1",
+      "name": "Add propType",
+      "description": "type string",
+      "startDate": "16.02.2022",
+      "deadlineDate": "24.02.2022",
+      "status": "",
+      "track": {"data": "25.02.2022", "node": "node Taks 2"}
+    }
+  ],
+  "subTasks": [
```

Comment:
Only they are users' tasks not subtasks. Don't mix up yourself

Reply:
ok, i deleted "track" form "task" in DB, and add method getTracks();

Reply:
```
export async function getTraks(userId) {
  const memberId = await userId;
  const response = await fetch(`${baseLocalURL}/users/?id=${memberId}`);
  const user = await response.json();
  const [{ tasksId }] = user;

  const traks = await Promise.all(
    tasksId.map(async (taskId) => {
      const responseTrack = await fetch(`${baseLocalURL}/tracks/?taskId=${taskId}`);
      const responseTask = await fetch(`${baseLocalURL}/tasks/?id=${taskId}`);
      const track = await responseTrack.json();
      const task = await responseTask.json();
      const [{ id, name }] = task;

      return { id, name, track };
    }),
  );

  return filterProgress(traks);
}

```

Reply:
now progress cteated from this
```
  "tracks": [
    {"id": "0", "taskId": "0", "data": "25.02.2022", "node": "node Taks 1"},
    {"id": "1", "taskId": "1", "data": "25.02.2022", "node": "node Taks 2"},
    {"id": "2", "taskId": "1", "data": "26.02.2022", "node": "node 2 for Taks 1"}
  ]
```

Reply:
I find taskId in task selected user and after find track. Should I put userId here?